docs/operations/hardening.md

```diff
@@ -107,7 +107,7 @@ kubelet_systemd_hardening: true
 # IP addresses, kubelet_secure_addresses allows you
 # to specify the IP from which the kubelet
 # will receive the packets.
-kubelet_secure_addresses: "localhost link-local {{ kube_pods_subnet }} 192.168.10.110 192.168.10.111 192.168.10.112"
+kubelet_secure_addresses: "localhost link-local {{ kube_pods_subnet }} {{ groups['all'] | map('extract', hostvars, ['ansible_host']) | join(' ') }}""
```

Comment:
Please keep in mind that `ansible_host` could potentially be set to a hostname or an FQDN.

Reply:
You're right. Also, as implemented in [Kubernetes roles](https://github.com/kubernetes-sigs/kubespray/blob/dd51ef6f96c4787f79484ff9452526f3aa91aa0a/roles/kubernetes/node/defaults/main.yml#L34), by default it sets to proper host nodes and this variable is an option to specify the IP list when there are multiple interfaces. So I'll close this PR.